tabpfn/scripts/transformer_prediction_interface.py

```diff
@@ -466,12 +501,13 @@ def get_preprocess(i):
         output_ = torch.cat([output_[..., class_shift_configuration:],output_[..., :class_shift_configuration]],dim=-1)
 
         #output_ = predict(eval_xs, eval_ys, style_, preprocess_transform_)
-        if not average_logits:
+        if not average_logits and not return_logits:
+            # transforms every ensemble_configuration into a probability -> equal contribution of every configuration
             output_ = torch.nn.functional.softmax(output_, dim=-1)
         output = output_ if output is None else output + output_
 
-    output = output / len(ensemble_configurations)
-    if average_logits:
+    output /= len(ensemble_configurations)
```

Comment:
I saw this do an inlace operation before which breaks gradient flow for some PyTorch version. Here is a little blog post on this issue. https://lernapparat.de/pytorch-inplace

So probably most save to go back to the old line